numalogic/connectors/utils/aws/config.py

```diff
@@ -0,0 +1,133 @@
+from dataclasses import dataclass, field
+from typing import Optional
+
+from numalogic.connectors.utils.enum import BaseEnum
+
+
+class DatabaseServiceProvider(BaseEnum):
+    """
+    A class representing the database service providers.
+
+    Attributes
+    ----------
+        rds (str): Represents the RDS (Relational Database Service) provider.
+        athena (str): Represents the Athena provider.
+
+    """
+
+    RDS = "rds"
+    ATHENA = "athena"
+
+
+class DatabaseTypes(BaseEnum):
```

Comment:
```suggestion
class DatabaseTypes(str, BaseEnum):
```
if you do this, then you can make a direct comparison